pom.xml

```diff
@@ -446,6 +446,12 @@
         <version>${project.version}</version>
       </dependency>
 
+	  <dependency>
```

Comment:
you used a tab character instead of a space - that's why in this view, the dependency tag on line 449 is a different indentation from the line above (447). This is not blocking the PR, but please fix if you get to this before the submit. Dipti knows how to change your IDE settings so that there won't be tab characters inserted anymore.